content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -324,16 +326,54 @@ single `.groovy` file which interact with each other, for example:
 ----
 // vars/acme.groovy
 def setName(value) {
-    this.name = value;
+    this.name = value
 }
 def getName() {
-    return this.name;
+    return this.name
 }
 def caution(message) {
     echo "Hello, ${this.name}! CAUTION: ${message}"
 }
 ----
 
+Note that in the above, `this.name` is not referring to a field, but to an entry in a `Script.binding`, so it is recommended to use a real class definition:
```

Comment:
`s/Note that in/In/`

Why is an entry in `Script.binding` a problem?   Add a quick comment on that.

Then Instead of "it is recommended",  you can just say "To avoid this,  define a `class`:".    
(Does the class have to have the same name as the file or can it be any name?)